### ISSUE TITLE
stylecop.json ignored by dotnet CLI

### ISSUE THREAD

**Issue by tillig** [DotNetAnalyzers/StyleCopAnalyzers#2142]
I'm using the dotnet CLI and the `additionalFile` argument [as outlined in the documentation](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/DotNetCli.md) to pass a relative path to the stylecop.json file.

My stylecop.json looks like this:

```json
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": {
    "documentationRules": {
      "xmlHeader": false
    },
    "indentation": {
      "indentationSize": 4,
      "tabSize": 4,
      "useTabs": true
    },
    "orderingRules": {
      "usingDirectivesPlacement": "outsideNamespace"
    }
  }
}
```

I have an empty ruleset file that looks like this:

```xml
<?xml version="1.0" encoding="utf-8"?>
<RuleSet Name="My Rules" Description="Description here." ToolsVersion="14.0">
  <!-- https://github.com/dotnet/roslyn/blob/master/docs/compilers/Rule%20Set%20Format.md -->
  <IncludeAll Action="Warning" />
  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
  </Rules>
</RuleSet>
```

My folder hierarchy looks like this:

```
/
  build/
    Analyzers.ruleset
    stylecop.json
  src/
    MyProject/
      project.json
```

As you can see, I have relative paths in place so all of my projects can use one ruleset and one stylecop.json.

My `additionalArguments` look like this:

`"additionalArguments": ["/ruleset:../../build/Analyzers.ruleset", "/additionalfile:../../build/stylecop.json"]`

Unfortunately, what I'm finding is the stylecop.json configuration seems to be ignored. For example, even though I have `useTabs: true` I still get tons of SA1027 errors telling me I shouldn't use tabs.

That said, I do see that I'm not getting any warnings about my namespace references being outside the namespace, and that's a configuration option, so maybe it's just the SA1027 rule not properly obeying configuration for tabs?

As a workaround I've disabled SA1027 in my ruleset for now.

**Comment by vweijsters** [comment]
If you are using StyleCop Analyzers v1.0.0, the useTabs settings is not available there. This setting will be available when 1.1.0-beta1 is released.

**Comment by tillig** [comment]
Ah, yeah, that's the issue. That wasn't clear from the docs. Thanks, I'll hold tight!
